pgx-examples/aggregate/aggregate.control

```diff
@@ -0,0 +1,5 @@
+comment = 'aggregate:  Created by pgx'
```

Comment:
Thought: perhaps we should have a URL that people can visit so they know what pgx is i they encounter it in their logs?

Reply:
I suggest opening an issue if you feel passionate about this in a general case! I think most users would set this to what their `Cargo.toml` description is!